requirements.txt

```diff
@@ -0,0 +1,7 @@
+backports.cached-property
+ops>=1.3.0,<2.0.0
+lightkube>=0.10.1,<1.0.0
+jsonschema
+pyyaml
+ops.manifest>=1.1.0,<2.0.0
+git+https://github.com/charmed-kubernetes/interface-kube-control.git@ca4f8f5aa8db8ff35d4de6ca52effee92d351de7#subdirectory=ops
```

Comment:
will need the correct commit-id from the squash-merge on [this](https://github.com/juju-solutions/interface-kube-control/pull/39)